lando_util/download.py

```diff
@@ -34,6 +35,12 @@ def download_files(dds_client, stage_items):
             click.echo("Writing file {}.".format(dest))
             with open(dest, 'w') as outfile:
                 outfile.write(source)
+        elif type == "unzip":
```

Comment:
I didn't catch this earlier, but it's odd that the `download_files` function does things like unzipping and writing files from local data. Wouldn't it be clearer to separate those into a different function/command?

Also, I find it confusing to use a variable called `type`, when type is a builtin class.

Reply:
I renamed the function and also the command to better describe what it does.
I removed the `unzip` item type and made it an option so the input file is more declarative.
It kind of felt like it was turning into a script.